spec/features/adventures/user_views_adventure_spec.rb

```diff
@@ -19,6 +19,10 @@
   "associated w/ a particular bucket lists by navigating to the bucket" \
   " list show page" do
     user = FactoryGirl.create(:user)
+    visit new_user_session_path
+    fill_in 'Email', with: user.email
+    fill_in 'Password', with: user.password
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.